src/objects/loadObj.js

```diff
@@ -7,7 +7,7 @@ const Objects = require('./objects.js');
 const OBJLoader = require("./loaders/OBJLoader.js");
 const MTLLoader = require("./loaders/MTLLoader.js");
 const FBXLoader = require("./loaders/FBXLoader.js");
-const GLTFLoader = require("./loaders/GLTFLoader.js");
```

Comment:
Looks like this file was copied from Three but maybe hasn't been updated? This solves the issue with the person model breaking when it is reloaded (and hits the threebox cache).

Kept this change minimal as not sure how much we want to diverge from the original.

Reply:
@josh-cloudscape any idea what was changed in the local version?

Reply:
@kbrownlees I assumed it was a copy / paste from Three without changes but could be wrong, will see what I can find.

Reply:
@josh-cloudscape that would be good, the other 5 loaders all seem like they are in the same boat? Feels like a nasty source of bugs if they have no been maintained against three.js correctly.

Reply:
@kbrownlees looks like it has been updated to r132, however it uses [this version](https://github.com/mrdoob/three.js/blob/r119/examples/js/loaders/GLTFLoader.js) (js) which seems to have a bug instead of [this version](https://raw.githubusercontent.com/mrdoob/three.js/r132/examples/jsm/loaders/GLTFLoader.js) (jsm) which doesn't. Not sure what the difference is between the two.

Reply:
I _think_ the js versions get generated from the other versions? Possibly the bug is caused by transpiling to an older es version?

Reply:
@josh-cloudscape interesting find! feels like we should just use the upstream version of all these loaders?

Reply:
I take it all back, that doesn't seem to be working for me today 😖 Not quite sure why. There seems to be some issues with that model, the box size is also strange and doesn't match the visual size. Might have to learn some blender and see if I can spot any issues. It does seem to have some extra things for animation in the model file.

Reply:
@josh-cloudscape yes there is, Jeff loves the thought of the models doing actions. https://github.com/cloudscapelabs/foreman.safesite-hq/commit/69971293348922f5ec1bb127d2b1af8a0b0bba02 was working at one point.